src/main/resources/xslt/cassandra/test-pom.xsl

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:pom="http://maven.apache.org/POM/4.0.0" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xalan="http://xml.apache.org/xslt" exclude-result-prefixes="pom xalan">
+
+    <xsl:output method="xml" indent="yes" xalan:indent-amount="2" />
+    <xsl:strip-space elements="*" />
+
+    <xsl:template match="@* | node()">
+        <xsl:copy>
+            <xsl:apply-templates select="@* | node()" />
+        </xsl:copy>
+    </xsl:template>
+
+    <!-- Force quarkus-cassandra-client-test.version property for any artifactId matching cassandra-quarkus-integration-tests -->
+    <xsl:template match="/pom:project/pom:dependencies/pom:dependency/pom:version">
+        <xsl:copy>
+            <xsl:if test="starts-with(../pom:artifactId/text(), 'cassandra-quarkus-integration-tests')"><xsl:apply-templates select="@*" />${quarkus-cassandra-client-test.version}</xsl:if>
+        </xsl:copy>
```

Comment:
Is this still needed?

Reply:
Yes, unless I'm misunderstanding the fundamentals of the platform generation.

The end goal is to be able to specify one version for the Cassandra artifacts that are listed in the BOM. And another for the test artifacts that we want to use. So this transform ensures that the tests artifacts are always set to the version that we specify in `quarkus-cassandra-client-test.version`.

Reply:
It'd be a bug if they aren't matching the version you configured in the `platformConfig`. Could you remove it and check whether it's really necessary?

Reply:
Yes, you are right. I've removed the transform entirely.

Apologies for the confusion.